src/index.html

```diff
@@ -2,13 +2,51 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta 
+    name="viewport" 
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+    rel="preconnect" 
+    href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+        class="moyoLogo" 
+        src="images/logo.png" 
+        alt="moyo_logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="linksContainer">
+          <li class="liClass">
+            <a 
+            class="links is-active" 
+            href="#"
+            >Apple</a>
+          </li>
+          <li class="liClass"><a class="links" href="#">Samsung</a></li>
+          <li class="liClass"><a class="links" href="#">Smartphones</a></li>
+          <li class="liClass">
+            <a 
+            data-qa="hover" 
+            class="links" 
+            href="#"
+            >Laptops & Computers</a>
+          </li>
```

Comment:
When line-wrapping, each continuation line should be indented 2 spaces from the original line. Also, attributes, should be indented with 2 spaces, and closing braket(>) should be on the new line
Please refer to styleguide and fix your formatting accordingly to it:
https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping